python/hsfs/statistics.py

```diff
@@ -19,14 +19,28 @@
 
 from hsfs import util
 from hsfs.split_statistics import SplitStatistics
+from hsfs.core.feature_descriptive_statistics import FeatureDescriptiveStatistics
 
 
 class Statistics:
     def __init__(
         self,
         commit_time,
+        row_percentage=100,
```

Comment:
This is becoming a fraction, should be 1

Reply:
Changed